packages/sdk-js/src/types/growthbook.ts

```diff
@@ -23,7 +23,12 @@ export type FeatureRule<T = any> = {
   weights?: number[];
   key?: string;
   hashAttribute?: string;
+  fallbackAttribute?: string;
   hashVersion?: number;
+  disableStickyBucketing?: boolean;
+  bucketVersion?: number;
+  minBucketVersion?: number;
+  excludeBlockedBucketUsers?: boolean;
```

Comment:
What are `disableStickyBucketing` and `excludeBlockedBucketUsers` used for?

Reply:
- `disableStickyBucketing` is an experiment-level kill switch. We don't support it yet in the UI, but do in the SDK.
- `excludeBlockedBucketUsers` controls the behavior of sticky bucketed users (removes them instead of rebuckets) when we block a variation. Again there is no support in the UI for variation blocking yet.

Reply:
I might get rid of the `excludeBlockedBucketUsers` part for now until we know for sure what data type will make sense when we do the UI side of things. For example, we might want to send down an array of blocked variations instead of just a  single boolean. Don't want to add things we aren't sure we'll actually need.

The kill switch, I'm fine leaving in if you want.

Reply:
sure, removed excludeBlockedBucketUsers. If the bucket variation key isn't found in the experiment, we now just re-bucket